src/bundle/Resources/translations/linkmanager.en.xliff

```diff
@@ -41,6 +36,11 @@
         <target state="new">Editing URL: %url%</target>
         <note>key: url.editing</note>
       </trans-unit>
+      <trans-unit id="51af0ce5242ede206403aac1116ee36ed8887d44" resname="url.editing.title">
+        <source>url.editing.title</source>
```

Comment:
The source should be replaced with proper translation. AFAIK, backend devs decided to change the approach of inserting translations in *.php files and now we have to write translations in `.xliff` manually.

Reply:
Actually, it is missing desc in twig.